src/mpyl/steps/deploy/bpm/__init__.py

```diff
@@ -13,18 +14,35 @@
 def deploy_to_cluster(
     logger: Logger, project_name: str, config: CamundaConfig
 ) -> Output:
-    docker_container = get_docker_container(config)
-    try:
-        deploy_diagram_to_cluster(logger, config, docker_container)
-    except DockerException:
-        return Output(
-            success=False,
-            message=f"Deploy BPMN diagrams to cluster for project {project_name} have one or more failures",
-            produced_artifact=None,
+    bpm_file_path = config.deployment_path.bpm_diagram_folder_path
+
+    for file_name in (
+        [fn for fn in os.listdir(bpm_file_path) if fn.endswith(".bpmn")]
```

Comment:
This seems a bit hacky? Can't we configure in the `project.yml` which files to deploy?

Reply:
No changes were made to this functionality, this is what is originally in the repository. Since this change is only about removing the dependency on Docker, could we let this slide for now and get back to this in a later PR?

Reply:
Boy scout rule etc, but ok